### ISSUE TITLE
NTR: Aplasia of the olfactory bulb and Aplasia of the olfactory tract

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4660]
**Preferred term label:**
Aplasia of the olfactory bulb
Aplasia of the olfactory tract

**Synonyms**
Absent olfactory bulb
Absent olfactory tract

**Definition (free text, please give PubMed ID)**
Aplasia (absence) of the olfactory bulb, which causes anosmia, a complete loss of the sense of smell.
Aplasia (absence) of the olfactory tract, which causes anosmia, a complete loss of the sense of smell.

Source: PMID:27863011

**Parent term (use hpo.jax.org/app)**
HP_0002977 'Aplasia/Hypoplasia involving the central nervous system'

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.omim.org/entry/300868?search=300868&highlight=300868#7

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-5208-3432

**Comment by nicolevasilevsky** [comment]
@pnrobinson I can add this if it looks okay

**Comment by pnrobinson** [comment]
I added one of the terms but wasn't able to figure out where to put the Aplasia (absence) of the olfactory tract, i.e., what parent term. Ideas?
